cmd/root.go

```diff
@@ -79,9 +77,13 @@ func getLatestReleaseInfo(info chan<- string) {
 
 func getOcClient() *occlient.Client {
 	client, err := occlient.New()
+	checkError(err, "")
+	return client
+}
+
+func checkError(err error, context string) {
```

Comment:
can we get some comments here.
It might be nice to explain for example what context is.

Reply:
done